src/python/lib/clusterfuzz/fuzz/engine.py

```diff
@@ -70,6 +70,19 @@ def name(self):
     """Get the name of the engine."""
     raise NotImplementedError
 
+  def fuzz_additional_processing_timeout(self, options):
+    """Return the maximum additional timeout in seconds for additional
+    operations in fuzz() (e.g. merging back new items).
+
+    Args:
+      options: A FuzzOptions object.
+
+    Returns:
+      An int representing the number of seconds required.
+    """
+    del options
```

Comment:
What is this needed for?

Reply:
This is the new recommended approach to dealing with unused arguments: 

https://google.github.io/styleguide/pyguide.html#214-decision